src/QMCWaveFunctions/BsplineFactory/createComplexSingle.cpp

```diff
@@ -36,10 +37,24 @@ BsplineReaderBase* createBsplineComplexSingle(EinsplineSetBuilder* e, bool hybri
   BsplineReaderBase* aReader = nullptr;
 
 #if defined(QMC_COMPLEX)
-  if (hybrid_rep)
-    aReader = new HybridRepSetReader<HybridRepCplx<SplineC2C<float>>>(e);
+#if defined(ENABLE_OFFLOAD)
+  if (useGPU == "yes")
+  {
+    if (hybrid_rep)
+    {
+      APP_ABORT("OpenMP offload has not been enabled with hybrid orbital representation!");
```

Comment:
Same comment as the non-single implementation. The code should use the CPU path in this case.

Reply:
No reason to need more user actions. Fallback to host code path.